Mathlib/RingTheory/Artinian.lean

```diff
@@ -490,16 +490,15 @@ variable (S : Submonoid R) (L : Type*) [CommRing L] [Algebra R L] [IsLocalizatio
 theorem localization_surjective : Function.Surjective (algebraMap R L) := by
   intro r'
   obtain ⟨r₁, s, rfl⟩ := IsLocalization.mk'_surjective S r'
-  obtain ⟨r₂, h⟩ : ∃ r : R, IsLocalization.mk' L 1 s = algebraMap R L r
-  swap
```

Comment:
Maybe `obtain; swap` should be replaced by `rsuffices` instead

Reply:
I tried using it and did not succeed. Be my guest to use it.

Reply:
It looks like `rsuffices` only supports a "stream of conciousness" syntax. More precisely,
```lean
rsuffices h : p a := b
```
is equivalent to
```lean
obtain h : p a := b
```
not something like
```lean
obtain h : p a; rotate_left; exact b
```

Reply:
@grunweg Could you please add `TODO`s in these cases? I guess, fixing `rsuffices` it out of scope for this PR, but let's have a reminder to swap these blocks back.

Reply:
Added a TODO.